### PR TITLE
Removes the EE ship.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
As agreed between Danse and the maintainers, any hostile-leaning offmap ships will have to wait until a better framework is coded and they are integrated into it.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Comment by NonQueueingMatt** [comment]
!review

**Review by mikomyazaki — CHANGES_REQUESTED** [review]
Gotta delete the access datum
```/var/const/access_ee_spy_ship = 205
/datum/access/ee_spy_ship
	id = access_ee_spy_ship
	access_type = ACCESS_TYPE_CENTCOM```
	
	And the stuff in the `map_shuttles` list.

**Review by mikomyazaki — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]

File: maps/away/ships/corporate_ship_ghostroles.dm [hunk 1]
```diff
@@ -65,85 +65,4 @@
 
 /obj/item/card/id/orion_ship
 	name = "orion express ship id"
-	access = list(access_orion_express_ship, access_external_airlocks)
```
Comment (JohnWildkins) [inline maps/away/ships/corporate_ship_ghostroles.dm:68]: don't delete this line

**Review by JohnWildkins — APPROVED** [review]

File: maps/away/ships/corporate_ship_ghostroles.dm (hunk 1 above)
Comment (Ben10083) [reply maps/away/ships/corporate_ship_ghostroles.dm:68]: did you mean to approve this?

**Comment by NonQueueingMatt** [comment]
unnecessary

**Pull request closed without merging**